.travis.yml

```diff
@@ -0,0 +1,19 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install: true
+install: true
+before_script: true
+script: true
+jobs:
+  include:
+    - stage: optional
+      script: ci/rpm7/build.sh -o -p $(pwd)
+    - stage: unit
+      before_script: docker run -t --rm -v $(pwd):/opt/fiware-orion fiware/orion-ci:rpm7 build -M unit
+      script: docker run -t --rm -v $(pwd):/opt/fiware-orion fiware/orion-ci:rpm7 build -IDu
+    - stage: functional
+      before_script: docker run -t --rm -v $(pwd):/opt/fiware-orion fiware/orion-ci:rpm7 build -M functional
+      script: docker run -t --rm -v $(pwd):/opt/fiware-orion fiware/orion-ci:rpm7 build -IDFf
```

Comment:
Doubt: how do travis execute the different stages? In sequence one by one (i.e. optional -> unit -> functional)? Or in parallel?

(Mabye a stupid question, but I don't know too much about travis details :)

Reply:
In this case: one by one.

Reply:
Thanks for clarifying! :)

NTC